test/xdpmp/miniport.c

```diff
@@ -996,7 +996,8 @@ MpReadConfiguration(
     Adapter->RxBufferLength = DEFAULT_RX_BUFFER_LENGTH;
     TRY_READ_INT_CONFIGURATION(ConfigHandle, RegRxBufferLength, &Adapter->RxBufferLength);
     if (Adapter->RxBufferLength < MIN_RX_BUFFER_LENGTH ||
-        Adapter->RxBufferLength > MAX_RX_BUFFER_LENGTH) {
+        Adapter->RxBufferLength > MAX_RX_BUFFER_LENGTH ||
+        !RTL_IS_POWER_OF_TWO(Adapter->RxBufferLength)) {
```

Comment:
This a new requirement?

Reply:
Yeah, in practice we've always used power-of-two buffers, and enforcing this allows us to bitmask rather than divide. I'd imagine most NICs have similar designs.